src/exports/CBL_EE_Exports.txt

```diff
@@ -64,18 +68,56 @@ CBLIndexUpdater_Value
 ### PRIVATE
 
 CBLCollection_IsIndexTrained
+CBLKeyPair_GenerateRSAKeyPair
 
 ### URLEndpointListener
 
 CBLListenerAuth_CreatePassword
 CBLListenerAuth_CreateCertificate
 CBLListenerAuth_Free
 CBLURLEndpointListener_Create
-CBLURLEndpointListener_Free
 CBLURLEndpointListener_Config
 CBLURLEndpointListener_Port
 CBLURLEndpointListener_Urls
 CBLURLEndpointListener_Status
 CBLURLEndpointListener_Start
 CBLURLEndpointListener_Stop
 
+### TLSIdentity
+
+kCBLCertAttrKeyCommonName
+kCBLCertAttrKeyPseudonym
+kCBLCertAttrKeyGivenName
+kCBLCertAttrKeySurname
+kCBLCertAttrKeyOrganization
+kCBLCertAttrKeyOrganizationUnit
+kCBLCertAttrKeyPostalAddress
+kCBLCertAttrKeyLocality
+kCBLCertAttrKeyPostalCode
+kCBLCertAttrKeyStateOrProvince
+kCBLCertAttrKeyCountry
+
+kCBLCertAttrKeyEmailAddress
+kCBLCertAttrKeyHostname
+kCBLCertAttrKeyURL
+kCBLCertAttrKeyIPAddress
+kCBLCertAttrKeyRegisteredID
+
+CBLKeyPair_RSAKeyPairWithCallbacks
+CBLKeyPair_RSAKeyPairWithPrivateKeyData
+CBLKeyPair_PublicKeyDigest
+CBLKeyPair_PublicKeyData
+CBLKeyPair_PrivateKeyData
+CBLCert_CertFromData
+CBLCert_CertNextInChain
+CBLCert_Data
+CBLCert_SubjectName
+CBLCert_SubjectNameComponent
+CBLCert_getValidTimespan
+CBLCert_PublicKey
+
+CBLTLSIdentity_Certificates
+CBLTLSIdentity_Expiration
+
+CBLTLSIdentity_SelfSignedCertIdentity
+CBLTLSIdentity_IdentityWithKeyPairAndCerts
```

Comment:
These new APIs are not listed here:

- CBLTLSIdentity_IdentityWithSecIdentity
- CBLTLSIdentity_IdentityWithLabel
- CBLTLSIdentity_deleteIdentityWithLabel
- CBLTLSIdentity_SelfSignedCertIdentityWithLabel

Maybe that's on purpose? 🤷

Reply:
Yes, on purpose. I will add them after they are implemented.